sdk/monitor/test-resources.json

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
I would be very happy if you decided to change this to a bicep template (`az bicep decompile -f test-resources.json  # outputs test-resources.bicep`) ;) In general it's a much simpler format that's easier to work with over time.

Reply:
I will address this right after the release since i'm running into a few issues - created an issue for this milestone here https://github.com/Azure/azure-sdk-for-python/issues/25213